Makefile

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
remove this file

Reply:
doh - can't believe I added it back again ;) sorry about that...

Reply:
@wwitzel3 I just pushed an updated PR without that pesky Makefile ;)